src/components/toolbar/PanelPreferencesToolbar.tsx

```diff
@@ -4,16 +4,28 @@ import { css } from '@emotion/react';
 import { Toolbar } from './Toolbar';
 
 interface PanelPreferencesToolbarProps {
+  title?: string;
   onClose?: () => void;
   onSave?: () => void;
 }
 
 const styles = {
+  container: css({
+    display: 'flex',
+    justifyContent: 'space-between',
+    borderBottom: '0.55px solid rgb(240, 240, 240)',
```

Comment:
`0.55px` seems like a magical value.

It doesn't work well with `1px`?

Reply:
I need to try it. It was always like that. when I implemented [PanelPreferencesToolbar.tsx](https://github.com/zakodium-oss/react-science/pull/633/files/480380937f3f382b250ebe0f99f6d98bded13063..e5481059da6338c15928fee55d36e018300311dc#diff-0117685563b69abe2dd16abb4d63a1a9994eb1bd279242649fa7001668243a83) I took the design directly from `nmrium`